src/actions/EventsActions.ts

```diff
@@ -23,3 +24,26 @@ export const getListOfEventsForHome = (
       });
   });
 };
+
+export const getEventById = (id: string) => {
```

Comment:
Maybe drop RESTful naming

Reply:
In this case that is a function I'm not sure if would look good:

`EventById(eventId)`

@e-azocar please help with your opinion

Reply:
Other options would be

- `eventById`
- `readEvent`
- `queryEventById`

Reply:
I like `eventById` and `queryEventById`

Reply:
I like `queryEventById`, but if you want to delete how would be the name? `mutationEvent`?
So, I prefer to use `get`, `delete`, `update` and `create`, this only for Javascript functions, I think the queries and mutations could be `NEW_USER_MUTATION`, `USER_BY_ID_QUERY`...

Reply:
Let's move this to any discussion to avoid blocking, it can be fixed in a refactor PR @Israel-Laguan @e-azocar

Reply:
I have created a discussion here to talk about it: https://github.com/Azordev/did-app/discussions/48